portfolio/src/main/webapp/style.css

```diff
@@ -120,6 +121,36 @@ img {
   font-family: "Courier New", Courier, monospace;
 }
 
+#comments {
+  margin-top: 100px;
+}
+
+.comment-list {
+  display: block;
+  list-style-type: none;
+  margin-left: 25%;
+  margin-right: 25%;
+  padding-left: 0px;
+  width: auto;
+}
+
+.comment-list > li > p {
```

Comment:
Great use of child combinators in CSS to apply different styles to the paragraph vs. span elements inside your comment list! I didn't know about child combinators until now; thanks for showing me a cool feature :)!

Reply:
Thanks! I didn't know about them either before this week